tests/Microsoft.Identity.Web.Test/WebAppExtensionsTests.cs

```diff
@@ -90,14 +91,19 @@ public void AddMicrosoftWebApp_WithConfigActionParameters(bool subscribeToDiagno
             services.AddDataProtection();
 
             new AuthenticationBuilder(services)
-                .AddMicrosoftWebApp(_configureOidcOptions, _configureMsOptions, _oidcScheme, _cookieScheme, subscribeToDiagnostics);
+                .AddMicrosoftWebApp(_configureOidcOptions, _configureMsOptions, _configureCookieOptions, _oidcScheme, _cookieScheme, subscribeToDiagnostics);
 
             var provider = services.BuildServiceProvider();
 
             // Assert configure options actions added correctly
             var configuredOidcOptions = provider.GetServices<IConfigureOptions<OpenIdConnectOptions>>().Cast<ConfigureNamedOptions<OpenIdConnectOptions>>();
             var configuredMsOptions = provider.GetServices<IConfigureOptions<MicrosoftIdentityOptions>>().Cast<ConfigureNamedOptions<MicrosoftIdentityOptions>>();
 
+#if DOTNET_CORE_31
+            var configuredCookieOptions = provider.GetServices<IConfigureOptions<CookieAuthenticationOptions>>().Cast<ConfigureNamedOptions<CookieAuthenticationOptions>>();
```

Comment:
Did you try to use IConfigureNamedOptions (interface) instead of an implementation?

Reply:
Yea, but the service is of type `IConfigureOptions` and interfaces don't have the `Action` property.

Reply:
did you see [this](https://github.com/AzureAD/microsoft-identity-web/pull/213/files#diff-36a4b8182e99fbf99b12e95c7a4e6dabR102), as an example? I would prefer to not have flags for .net 3.1/net 5 is possible.

Reply:
Hmm, yea, I could just try to resolve the service for `IConfigureOptions<CookieAuthenticationOptions>` but that will only tell us that some cookie options were added and not whether our custom cookie action was added.

We might still need flags in general, like for #234 fix.